pkg/logger/setting.go

```diff
@@ -66,24 +68,16 @@ func Init(cfg Logging) (err error) {
 
 // getLogger initializes a root logger
 func getLogger(cfg Logging) (*Logger, error) {
-	// parse logging level
-	level := zap.NewAtomicLevelAt(zapcore.InfoLevel)
-	if err := level.UnmarshalText([]byte(cfg.Level)); err != nil {
+	lvl, err := zerolog.ParseLevel(cfg.Level)
+	if err != nil {
 		return nil, err
 	}
-	var encoderConfig zap.Config
+	w := io.Writer(os.Stderr)
 	switch cfg.Env {
 	case "dev":
-		encoderConfig = zap.NewDevelopmentConfig()
-		encoderConfig.EncoderConfig.EncodeLevel = zapcore.CapitalColorLevelEncoder
-		break
+		w = zerolog.ConsoleWriter{Out: os.Stderr}
```

Comment:
Would you use https://github.com/rs/zerolog#pretty-logging to format dev log like: 
`// Output: 2006-01-02T15:04:05Z07:00 | INFO  | ***Hello World**** foo:BAR`

Reply:
Fixed.

```
2021-07-01T16:32:43+08:00 | DEBUG | ***register flags**** module:STANDALONE name:DATABASE registered:1 total:3
```